contracts/Ante_alETHSupplyTest.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.7.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./interfaces/AnteTest.sol";
+
+// Ante Test to check alETH supply never exceeds amount of ETH locked in Alchemix
+contract Ante_alETHSupplyTest is AnteTest("alETH doesn't exceed ETH locked in Alchemix") {
+    // https://etherscan.io/address/0x0100546F2cD4C9D97f798fFC9755E47865FF7Ee6
+    address public constant alETHAddr = 0x0100546F2cD4C9D97f798fFC9755E47865FF7Ee6;
+    // https://etherscan.io/token/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2
+    address public constant WETHAddr = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+    // https://etherscan.io/address/0xa258C4606Ca8206D8aA700cE2143D7db854D168c
+    address public constant yvWETHAddr = 0xa258C4606Ca8206D8aA700cE2143D7db854D168c;
+
+    address public constant TransmuterAddr = 0x45f81eF5F2ae78f49851f7A62e4061FF54Ff674B;
+    address public constant AlchemistAddr = 0x6B566554378477490ab040f6F757171c967D03ab;
+    address public constant AlchemistYVAAddr = 0xEBA649E0010818Aa4321088D34bD6162d65E7971;
+    address public constant TransmuterYVAddr = 0x54dc35eb8c2E2E20f3657Af6F84cd9949C08CF38;
+
+    IERC20 public alETHToken = IERC20(alETHAddr);
+    IERC20 public WETHToken = IERC20(WETHAddr);
+    IERC20 public yvWETHToken = IERC20(yvWETHAddr);
+
+    constructor () {
+        protocolName = "Alchemix";
+        testedContracts = [alETHAddr];
+    }
+    
+    function checkTestPasses() public view override returns (bool) {
+        uint TransmuterVL = WETHToken.balanceOf(TransmuterAddr) / 1e18;
+        uint AlchemistVL = WETHToken.balanceOf(AlchemistAddr) / 1e18;
+        uint AlchemistYVAVL = yvWETHToken.balanceOf(AlchemistYVAAddr) / 1e18;
+        uint TransmuterBYVAVL = yvWETHToken.balanceOf(TransmuterYVAddr) / 1e18;
+        uint TotalValueLocked = TransmuterVL + AlchemistVL + AlchemistYVAVL + TransmuterBYVAVL;
+        uint TotalSupply = alETHToken.totalSupply() / 1e18;
```

Comment:
From our lead dev:
Doubt these calculations will overflow, though it's standard practice to use SafeMath to explicitly revert on under/overflow in arithmetic operations. You can also use Solidity 0.8.x where the reversion behavior is default.

Reply:
Thanks for noticing this, I'll upgrade to 0.8.0.